ui/packages/shared/profile/src/ProfileIcicleGraph/IcicleGraphArrow/index.tsx

```diff
@@ -106,50 +126,55 @@ export const IcicleGraphArrow = memo(function IcicleGraphArrow({
 
   const currentSearchString = (selectQueryParam('search_string') as string) ?? '';
   const {compareMode} = useProfileViewContext();
-  const isColorStackLegendEnabled = selectQueryParam('color_stack_legend') === 'true';
+  // const isColorStackLegendEnabled = selectQueryParam('color_stack_legend') === 'true';
```

Comment:
perhaps we're ready to remove this setting?

Reply:
mean't to delete this, must have missed this.